### DISCUSSION TITLE
Geometry Drawing

### DISCUSSION THREAD

**Discussion by tonehaven** [edwardkort/WWIDesigner#111] [NAF Study] [answered]
Thanks so much for sharing such a wonderful program and an obviously huge amount of time and effort.  I'm a novice to WWIDesigner and to flutes.  So, the terminology is confusing me a bit.  I've seen the cross-sectional images of the NAF and understand the basic physics and geometry, but the descriptions of the specific technical NAF instrument values in WWID that are in the wiki glossary are not quite sufficient for me to confidently know that I understand the technical geometry enough to execute a build.

Are there any technical drawings that define the terms, particularly the fipple/mouthpiece and the reference point for all the distances.  I've been hunting around the Wiki and haven't found one yet.

Thanks.

**Comment by edwardkort** [comment, marked as answer]
Hello,

The members of the development team address WIDesigner documentation in different fashions. Burton (the Whistle, Flute, and Reed Studies, as well as supplemental functionality) uses wiki pages exclusively. I (NAF Study and general program usage) mostly use primarily video tutorials (see [WIDesigner Video Tutorials](https://github.com/edwardkort/WWIDesigner/wiki/WIDesigner-Video-Tutorials)).

In particular, the NAF instrument model (usage and parameter definitions) is covered in the video: [WIDesigner Instrument Representation](https://www.youtube.com/watch?v=7sdnDpqy278&feature=youtu.be).

Cheers,
Edward Kort
Wood Wind
